src/Interface/Modules/Render/Tests/SRInterfaceTests.cc

```diff
@@ -69,5 +69,5 @@ TEST(SRInterfaceTest,
 {
   SRInterface srinterface;
 
-  srinterface.doFrame(0, 50);
+  srinterface.doFrame(50);
```

Comment:
What is the 50?

Reply:
Now that’s a magic number that we should label. It’s the milliseconds in between frames, more or less. I’ll have the graphics interns change it.

Reply:
This test says it renders an empty frame which isn't quite true. The 50 milliseconds as input is the same thing that caused the flicker every 6 clicks. It might work if we change it to 300, which we could just make a function to render an empty frame instead.

Reply:
I think "empty" here meant nothing was piped into ViewScene in this test, so there would be nothing to show...other than axes I guess.

Reply:
Since the axis is loaded by the promise fulfillment system we aren't guaranteed it will exist (from my experience it generally won't). What I think this test is doing is just bootstrapping the core. Since there is no geometry the rendering step should be skipped. If this wasn't the case I'd say we should remove this since otherwise, we'd be rendering to a random context since the doFrame code doesn't expect the right context is already bound. It shouldn't matter what the number we execute with is setting it to 0 should be fine especially since we already have a function for that